python/tvm/driver/tvmc/common.py

```diff
@@ -31,6 +33,26 @@
 # pylint: disable=invalid-name
 logger = logging.getLogger("TVMC")
 
+TVM_TARGETS = [
```

Comment:
@junrushao1994 Just wanted to mention that we’re working on exposing this API to the Python layer, but it is just the case the follows the classic “this could be a separate PR” if posted together here.

Reply:
This is https://github.com/apache/tvm/pull/7427

Reply:
As #7427 is merged, the new version makes use of the new API call.